### PR TITLE
Update plan-auto-attendant-call-queue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Adding agent level additional requirements for conference/transfer of CQ calls recieved via Direct Routing.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/plan-auto-attendant-call-queue.md [hunk 1]
```diff
@@ -69,7 +69,10 @@ To configure auto attendants and call queues, you need the following resources:
 - At least one [Microsoft service number](getting-service-phone-numbers.md), direct routing number, or a hybrid number for each resource account that you want to be directly dialable
  - The service number may be a toll or toll-free number
 
-Agents who receive calls from the call queues must be Enterprise Voice enabled online or on-premise users. 
+Agents who receive calls from the call queues must be Enterprise Voice enabled online or on-premise users. In addition, if the call queues are using Direct Routing numbers, agents who need to conference or transfer calls also require an:
```
Comment (JohanFreelancer9) [inline Teams/plan-auto-attendant-call-queue.md:72]:
```suggestion
Agents who receive calls from the call queues must be Enterprise Voice enabled online or on-premise users. In addition, if the call queues are using Direct Routing numbers, agents who need to conference or transfer calls also require:
```

File: Teams/plan-auto-attendant-call-queue.md [hunk 2]
```diff
@@ -69,7 +69,10 @@ To configure auto attendants and call queues, you need the following resources:
...
+
+- Online Voice Routing Policy assigned if the call queue uses transfer mode
```
Comment (JohanFreelancer9) [inline Teams/plan-auto-attendant-call-queue.md:74]:
```suggestion
- An online voice routing policy assigned if the call queue uses transfer mode
```

File: Teams/plan-auto-attendant-call-queue.md [hunk 3]
```diff
@@ -69,7 +69,10 @@ To configure auto attendants and call queues, you need the following resources:
...
+- Audio Conferencing license or Online Voice Routing Policy assigned if the call queue uses conference mode
```
Comment (JohanFreelancer9) [inline Teams/plan-auto-attendant-call-queue.md:75]:
```suggestion
- An Audio Conferencing license or online voice routing policy assigned if the call queue uses conference mode
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**